packages/check/src/main.rs

```diff
@@ -120,7 +120,7 @@ fn check_contract(
         Logger::Off
     };
     // Check wasm
-    check_wasm_with_logs(&wasm, available_capabilities, logs)?;
+    check_wasm_with_limits(&wasm, available_capabilities, &WasmLimits::default(), logs)?;
```

Comment:
Ah, here we need the option to specify the limits in cosmwasm-check. Otherwise the contracts can't be checked anymore for non-default environments.

Ideally we can copy/paste a config between cosmwasm-check and the chain

Reply:
True. We could add a cli command or even a query to wasmd to get the config and then you can pass that to cosmwasm-check using a `--limits` flag or something like that.